### PR TITLE
better build-logs error messages

### PR DESCRIPTION
**Pull request by bparees**
fixes https://github.com/openshift/origin/issues/2541

### CODE REVIEW

**Comment by bparees** [comment]
@csrwng ptal.

File: pkg/cmd/cli/cmd/buildlogs.go [hunk 1]
```diff
@@ -44,6 +45,15 @@ func NewCmdBuildLogs(fullName string, f *clientcmd.Factory, out io.Writer) *cobr
 					fmt.Fprintf(out, msg)
 					os.Exit(1)
 				}
+				if err.Status().Code == http.StatusNotFound {
+					switch err.Status().Details.Kind {
+					case "build":
+						fmt.Fprintf(out, "The build %s could not be found.  Therefore build logs cannot be retrieved.\n", err.Status().Details.Name)
+					case "pod":
+						fmt.Fprintf(out, "The pod %s for the build %s could not be found.  Therefore build logs cannot be retrieved.\n", err.Status().Details.Name, args[0])
```
Comment (csrwng) [inline pkg/cmd/cli/cmd/buildlogs.go]: s/The pod %s for the build %s/The pod %s for build %s/
I think sounds slightly better.

Comment (bparees) [inline pkg/cmd/cli/cmd/buildlogs.go]: changed.

**Comment by csrwng** [comment]
LGTM [merge]

**Pull request merged**